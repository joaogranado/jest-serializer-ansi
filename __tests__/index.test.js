const serializer = require('../');

it('should strip ansi codes', () => {
  expect.addSnapshotSerializer(serializer);

  expect('foo\u001B[4mcake\u001B[0m').toMatchSnapshot();
});
