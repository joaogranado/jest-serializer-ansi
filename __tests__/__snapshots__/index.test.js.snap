// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should strip ansi codes 1`] = `foocake`;
